db/migrate/20170725153927_create_data_requests.rb

```diff
@@ -0,0 +1,16 @@
+class CreateDataRequests < ActiveRecord::Migration[5.1]
+  def change
+    create_table :data_requests, id: :uuid do |t|
+      t.integer :user_id
+      t.integer :workflow_id
```

Comment:
On a branch I would normally just edit the migration and `bin/rails db:migrate:redo VERSION=20170725153927`.

I think for now we should also just stick to DataRequests that actually have a workflow_id. I can see the use for ones that are scoped to a user within a workflow, but all data from a user across workflows might be premature optimization. YAGNI.